recipes/holoviews-1.5.0/meta.yaml

```diff
@@ -0,0 +1,42 @@
+package:
+  name: holoviews
+  version: 1.5.0
+
+source:
+  path: .
+
+build:
+  number: 0
+  script: python setup.py install
```

Comment:
You can remove both `build.sh` and `bld.bat`. Latest conda will ready this line and build it without the build scripts. Also, since holoviews seems to be using `setuptools` you can use:

`script: python setup.py install --single-version-externally-managed --record record.txt` there.